Data/Scripts/GardenConquest/Core/CommandProcessor.cs

```diff
@@ -21,20 +21,72 @@ class CommandProcessor {
 
 		private static Logger s_Logger = null;
 
-		// TODO: rest of this
-		private static string s_AboutText =
-			"Welcome to Garden Conquest\n\n" +
-			"In GC each grid must have a Hull Classifier ... MORE";
-
 		private static string s_HelpText =
-			"Chat commands:\n\n" +
-			"/gc about - Show basic mod info\n" +
+			"Garden Conquest is a new, open source, Conquest-type mod for " +
+			"Space Engineers that introduces area-control dynamics for " +
+			"combat-focused survival servers.\n\n" +
+			"If you're new to GC, check out the help subtopics below.\n\n" +
+			"Chat Commands:\n" +
 			"/gc help - Show this screen\n" +
-			"/gc help [category] - Show help for a category.  Categories are:\n" +
-			"    limits - Information on grid class limitations\n";
+			"/gc help [category] - Show help for a category.  Categories:\n" +
+			"        classes     - Ship Classes and their limits\n" +
+			"        classifiers - Hull Classifier blocks\n" +
+			"        cps            - Control Points\n" +
+			"        licenses    - Ship License components\n" +
+			"/gc fleet - Information on your faction's fleet \n" +
+			"/gc fleet remove \"Ship Name\"- Disown a ship";
+
+		private static string s_HelpClassText =
+			"Classes:\n\n" +
+			"'5 L' means the Classifier costs 5 Ship Licenses\n" +
+			"'Small' and 'Large' below refer to Ship sizes.\n" +
+			"Block, turret, etc counts are maximums.\n\n";
+
+		private static string s_HelpClassifiersText =
+			"Hull Classifiers are new blocks that each correspond to one " +
+			"of the various classes.\n\n" +
+			"Classifiers must be fully built and powered to designate your " +
+			"ship as a member of its class.\n\n" +
+			"Every Ship/Station must be classified, even the \"Unlicensed\" " +
+			"ones. Unlicensed classifiers do not require Ship Licenses " +
+			"to build, but they have relatively low block limits.\n\n" +
+			"If a ship goes unclassified for 2 hours, it will become a " +
+			"Derelict and slowly deteriorate until it either is reclassified " +
+			"by a player or disappears entirely.\n\n" + 
+			"For more info on Classes, type \"/gc help classes\"";
+
+		private static string s_HelpCPsText1 =
+			"Control Points are areas of the map you can hold and control. " +
+			"Each CP has a Radius, Reward, and Round Time. On this server, " +
+			"these are:\n\n";
+
+		private static string s_HelpCPsText2 =
+			"\n\nIn order to win a 15 minute round, you or your faction must " +
```

Comment:
Would you mind including for now and then I can adjust once I see how you fill out the info for the other configuration options?